File: test_login.py
import urllib.request
import json

url = "https://backend-production-83b1.up.railway.app/api/auth/login"
data = json.dumps({"username": "admin@kamlog.cm", "password": "admin123"}).encode('utf-8')
headers = {'Content-Type': 'application/json'}
req = urllib.request.Request(url, data=data, headers=headers, method='POST')

try:
    with urllib.request.urlopen(req, timeout=10) as response:
        print("Status:", response.status)
        print("Response:", response.read().decode('utf-8'))
except urllib.error.HTTPError as e:
    print("HTTP Error Status:", e.code)
    print("Error Response:", e.read().decode('utf-8'))
except Exception as e:
    print("Exception:", e)

File: kamlog-frontend/test_login.js
const axios = require('axios');

async function testLogin() {
  const BASE_URL = 'https://backend-production-83b1.up.railway.app';
  console.log("Starting login test to:", BASE_URL);
  
  try {
    const response = await axios.post(`${BASE_URL}/api/auth/login`, {
      username: 'admin@kamlog.cm',
      password: 'admin123'
    }, {
      headers: { 'Content-Type': 'application/json' }
    });
    
    console.log("Login Success! Response data:", response.data);
  } catch (error) {
    if (error.response) {
      console.error("Login Failed! Status:", error.response.status);
      console.error("Response data:", error.response.data);
    } else {
      console.error("Network Error:", error.message);
    }
  }
}

testLogin();
